luxtronik_firmware.sh

```diff
@@ -11,19 +11,18 @@ echo "https://www.heatpump24.com/software/fetchSoftware.php?softwareID=$software
 filename=$(curl -s -I "https://www.heatpump24.com/software/fetchSoftware.php?softwareID=$softwareID" | grep -oP 'wp2?reg[-.][VF]\d+.\d+.\d+')
 echo "Latest firmware is $filename"
 
-if [[ "$filename" == *"wpreg."*  ]]; then
-    curl -s "https://www.heatpump24.com/software/fetchSoftware.php?softwareID=$softwareID" --output "$filename"
-    echo "Downloaded as $filename"
-    echo "Firmware $filename can be decompressed with python via \"./extract_V1_firmware.py $filename\"."
-    exit
+if [[ "$filename" == *"wpreg."* ]]; then
+	curl -s "https://www.heatpump24.com/software/fetchSoftware.php?softwareID=$softwareID" --output "$filename"
+	echo "Downloaded as $filename"
+	echo "Decompress downloaded firmware"
+	python extract_V1_firmware.py $filename
```

Comment:
For me, `python` is `python2`. Should we use just `./extract_V1_firmware.py $filename`?